src/v2/components/MessageList/TypingIndicator.tsx

```diff
@@ -24,22 +20,21 @@ import type {
 
 const styles = StyleSheet.create({
   container: {
-    alignItems: 'flex-end',
+    alignItems: 'center',
+    backgroundColor: '#FCFCFCE6',
     flexDirection: 'row',
+    height: 24,
     justifyContent: 'flex-start',
   },
+  loadingDots: {
+    marginLeft: 8,
+  },
   typingText: {
-    marginLeft: 10,
+    color: '#7A7A7A',
```

Comment:
I've seen this 7A color in a few places within the new designs. should we just put this within the default theme colors and use it from there?

Reply:
I think we need to do a big color pass through in Q1